src/settings/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { Settings } from '@folio/stripes/smart-components';
+
+import GeneralSettings from './general-settings';
```

Comment:
It is a good idea to name the component files with the pascal notation e.g `GeneralSettings`. This way is used in the majority of UI projects. I would recommend for other files in the project using the camelCase notation.
```suggestion
import GeneralSettings from './GeneralSettings';
```

Reply:
Fixed